packages/k8s/k3s/build.yaml

```diff
@@ -0,0 +1,31 @@
+requires:
+- name: "toolchain-go"
+  category: "development"
+  version: ">=0"
+env:
+  - INSTALL_K3S_BIN_DIR="/usr/bin"
+  - INSTALL_K3S_SELINUX_WARN=true
+  - INSTALL_K3S_SKIP_START="true"
+  - INSTALL_K3S_SKIP_ENABLE="true"
+  - INSTALL_K3S_SKIP_SELINUX_RPM="true"
+prelude:
+{{$arch:=(default .Values.arch "amd64")}}
```

Comment:
could this be the issue? After all this is a templating instruction so the prelude will be empy